pages/profile.js

```diff
@@ -0,0 +1,7 @@
+export default function profile() {
+  return (
+    <>
+      <h1>Personal Profile</h1>
+    </>
+  );
+}
```

Comment:
usually we only want to implement pages and components when the user can already use them. For now, its no problem, but keep in mind for future pages/components :)

Reply:
ok)